scipy/stats/tests/test_stats.py

```diff
@@ -4804,7 +4804,7 @@ def test_ttest_ind_permutation_check_inputs(self):
             stats.ttest_ind(self.a2, self.b2, permutations=1.5)
         with assert_raises(ValueError, match="'hello' cannot be used"):
             stats.ttest_ind(self.a, self.b, permutations=1,
-                            random_state='hello')
+                            random_state='hello', axis=1)
```

Comment:
The default `axis=0` is invalid for `self.a` and `self.b`, but the errors were raised in a different order before, so we didn't notice.